src/SprykerEco/Shared/PunchoutCatalogs/Transfer/punchout_catalog.transfer.xml

```diff
@@ -24,6 +24,20 @@
         <property name="createdAt" type="string" />
     </transfer>
 
+    <transfer name="PunchoutCatalogEntryPoint">
+        <property name="store" type="Store" />
+        <property name="url" type="string" />
```

Comment:
Where this field is used?

Reply:
It used here https://github.com/spryker-eco/punchout-catalogs/pull/9/files#diff-714fed64a6f047a764ace2851bb4a035